libraries/Wire/Wire.cpp

```diff
@@ -0,0 +1,57 @@
+#include "Wire.h"
+
+void arduino::ZephyrI2C::begin() {
+  // const struct device *dev = DEVICE_DT_GET_ANY(adi_adxl372);
+  *i2c_dev = DEVICE_DT_GET(DT_NODELABEL(i2c0));
+  printk("IIC begin\n");
+}
+
+void arduino::ZephyrI2C::begin(uint8_t slaveAddr) {
+
+}
+
+void arduino::ZephyrI2C::end() {}
+
+void arduino::ZephyrI2C::setClock(uint32_t freq) {}
+
+void arduino::ZephyrI2C::beginTransmission(uint8_t address) { // TODO for ADS1115
```

Comment:
Needs to create the `bus` with the i2c device + device address, like this https://github.com/zephyrproject-rtos/zephyr/blob/f9d69c34c9d003e99a026ee2a0da9c634e9598c7/drivers/adc/adc_ads1x1x.c#L153
```
struct i2c_dt_spec bus;
```

Reply:
sets `_addr` and reset `txBuf`